existential-vs-universal-quantification/article.md

```diff
@@ -0,0 +1,194 @@
+# Existential vs universal quantification
+
+## Introduction
+This article gives brief information about one of the most common extension in Haskell projects - `ExplicitForAll`
+and syntax constructions that this extension brings to language.
+Stay tuned if you want to know how to create lists of differently typed variables, how to add complex constraints and more!
+
+From type theory we know about the existence of two quantifiers - `$\forall$` and `$\exists$`.
+The former means: `"For every term x the statement is true"`, the later: `"There is at least one term x for which this statement is true"`.
+
+Since Haskell is based on type theory there is a place for these quantifiers in language syntax, however, in Haskell we are talking not about some terms and statements, but about types.
+By default, all type variables are bound by a `forall` quantifier, which means that every type, that passes constraints, can be used inside this function/type declaration.
```

Comment:
> every type, that passes constraints, can be used inside this function/type declaration

This might be slightly misleading, as it could give the impression that the function is free to choose what the type `a` can be, inside it. But AFAIU, you cannot take this argument, and say, use it as a `String`, or an `Int` or any other concrete type.